OpenAPI/kiota/using.md

```diff
@@ -468,48 +485,73 @@ kiota info -l CSharp --json
   "dependencies": [
     {
       "name": "Microsoft.Kiota.Abstractions",
-      "version": "1.6.1"
+      "version": "1.6.1",
+      "type": "abstractions"
     },
     {
       "name": "Microsoft.Kiota.Http.HttpClientLibrary",
-      "version": "1.2.0"
+      "version": "1.2.0",
+      "type": "http"
     },
     {
       "name": "Microsoft.Kiota.Serialization.Form",
-      "version": "1.1.0"
+      "version": "1.1.0",
+      "type": "serialization"
     },
     {
       "name": "Microsoft.Kiota.Serialization.Json",
-      "version": "1.1.1"
+      "version": "1.1.1",
+      "type": "serialization"
     },
     {
       "name": "Microsoft.Kiota.Authentication.Azure",
-      "version": "1.1.0"
+      "version": "1.1.0",
+      "type": "authentication"
     },
     {
       "name": "Microsoft.Kiota.Serialization.Text",
-      "version": "1.1.0"
+      "version": "1.1.0",
+      "type": "serialization"
     },
     {
       "name": "Microsoft.Kiota.Serialization.Multipart",
-      "version": "1.1.0"
+      "version": "1.1.0",
+      "type": "serialization"
+    },
+    {
+      "name": "Microsoft.Kiota.Bundle",
+      "version": "1.1.0",
+      "type": "bundle"
     }
   ],
   "clientClassName": "",
   "clientNamespaceName": ""
 }
 ```
 
-### Optional Parameters
+### `--dependency type (--dt)`
 
-The info command accepts optional parameters commonly available on the other commands:
+Since: 1.18.1
 
-- [--openapi](#--openapi--d)
-- [--clear-cache](#--clear-cache---cc)
-- [--log-level](#--log-level---ll)
-- [--language](#--language--l)
-- [--version](#--version--v)
-- [--search-key](#--search-key--k)
+The type of dependencies to display when used in combination with the language option. Does not impact the json output. Accepts multiple values. Default empty.
+
+```bash
+kiota info --language <language> --dependency-type authentication
+```
+
+Accepted values:
+
+- Abstractions dependencies define the core concepts of the language. Required at build time.
+- Authentication dependencies implement authentication providers. Optional at runtime.
+- Additional dependencies are required in addition to the abstractions or bundle. Required at build time.
+- Bundle dependencies include abstractions, serialization and HTTP dependencies for simpler management.
+- HTTP dependencies implement the request adapter with a specific HTTP client. Required at runtime.
+- Serialization dependencies implement serialization and deserialization for a given format. Required at runtime.
```

Comment:
Should we have a table with the actual enum name and then a description? It would make it clearer

Reply:
updated